docker-compose/docker-compose.edge.yml

```diff
@@ -1,134 +1,77 @@
 version: '3.2'
 services:
-  mosquitto-remote:
+  mosquitto:
     build:
-      context: ${IOTEA_PROJECT_DIR}/docker/mosquitto
-      dockerfile: Dockerfile.${ARCH}
+      context: ../iot-event-analytics/docker/mosquitto
+      dockerfile: Dockerfile.${ARCH-amd64}
       args:
         - HTTP_PROXY=${DOCKER_HTTP_PROXY}
         - HTTPS_PROXY=${DOCKER_HTTPS_PROXY}
-        - MQTT_PORT=1884
-    ports:
-      - '1884:1884'
-    volumes:
-      - type: bind
-        source: "${CONFIG_DIR}/mosquitto/remote"
-        target: "/mosquitto/config"
-  mosquitto-local:
-    build:
-      context: ${IOTEA_PROJECT_DIR}/docker/mosquitto
-      dockerfile: Dockerfile.${ARCH}
-      args:
-        - HTTP_PROXY=${DOCKER_HTTP_PROXY}
-        - HTTPS_PROXY=${DOCKER_HTTPS_PROXY}
-    depends_on:
-      - mosquitto-remote
     ports:
       - '1883:1883'
     volumes:
       - type: bind
-        source: "${CONFIG_DIR}/mosquitto/local"
+        source: "${CONFIG_DIR}/mosquitto"
         target: "/mosquitto/config"
-  configmanager:
+  iotea:
```

Comment:
iotea-platform? (because it is 'only' the platform) (same for the other compose files)

Reply:
Only one file contains the platform now, since they were separated by component